contracts/src/v0.8/dev/vrf/VRFCoordinatorV2Plus.sol

```diff
@@ -348,7 +347,7 @@ contract VRFCoordinatorV2Plus is VRF, SubscriptionAPI {
 
   function getRandomnessFromProof(
     Proof calldata proof,
```

Comment:
Would making this memory help or would compilation fail due to the slicing stuff?

Reply:
I can check that out.